### PR TITLE
Add logging info for the debugger

### PR DESCRIPTION
**Pull request by michelleangela**
Add logging info for the debugger.
Content from https://github.com/microsoft/vscode-cpptools/blob/master/Documentation/FAQs.md#how-to-enable-logging

### CODE REVIEW

**Comment by michelleangela** [comment]
@pieandcakes , @WardenGnaw , please review.

File: docs/cpp/enable-logging-cpp.md [hunk 1]
```diff
@@ -8,14 +8,13 @@ DateApproved: 07/25/2019
 MetaDescription: How to enable logging in the Visual Studio Code C/C++ extension
 ---
 # C/C++ extension logging
-
-If you are experiencing a problem that we are unable to diagnose based on information in your issue report, we might ask you to enable logging and send us your logs.
+Logging information is available for the language server and debugger. If you are experiencing a problem that we are unable to diagnose based on information in your issue report, we might ask you to enable logging and send us your logs.
 
 Logging information is delivered directly to the Output window in Visual Studio Code.
```
Comment (pieandcakes) [inline docs/cpp/enable-logging-cpp.md]: Need to change this to point to the Output window for the language service and debugConsole for the debugger.

File: docs/cpp/enable-logging-cpp.md [hunk 2]
```diff
@@ -28,3 +27,38 @@ To turn on full logging for an issue report, follow these steps:
 1. Select the "C/C++" option in the log filter selector:
 
    ![Log filter selector](images/cpp/log-filter-selector.png)
+
+## Enable logging for the debugger
```
Comment (pieandcakes) [inline docs/cpp/enable-logging-cpp.md]: Can you change this to say "for the debug adapter"

File: docs/cpp/enable-logging-cpp.md [hunk 3]
```diff
@@ -8,14 +8,13 @@ DateApproved: 07/25/2019
 MetaDescription: How to enable logging in the Visual Studio Code C/C++ extension
 ---
 # C/C++ extension logging
-
-If you are experiencing a problem that we are unable to diagnose based on information in your issue report, we might ask you to enable logging and send us your logs.
+Logging information is available for the language server and debugger. If you are experiencing a problem that we are unable to diagnose based on information in your issue report, we might ask you to enable logging and send us your logs.
```
Comment (pieandcakes) [inline docs/cpp/enable-logging-cpp.md]: Change to say "for the language server and debug adapter"

**Comment by pieandcakes** [comment]
Thanks for submitting this!

**Comment by gregvanl** [comment]
Added `don't merge` since under review

**Comment by michelleangela** [comment]
Adding @bobbrow

File: docs/cpp/enable-logging-cpp.md (hunk 2 above)
Comment (michelleangela) [reply docs/cpp/enable-logging-cpp.md]: Are there any other places where the "debugger" should be referred to as "debug adapter"?

File: docs/cpp/enable-logging-cpp.md [hunk 4]
```diff
@@ -28,3 +27,38 @@ To turn on full logging for an issue report, follow these steps:
 1. Select the "C/C++" option in the log filter selector:
 
    ![Log filter selector](images/cpp/log-filter-selector.png)
+
+## Enable logging for the debug adapter
+Enabling logging for the debug adapter will show communication information between VS Code and our extension and between our extension and the debug adapter.
+
+### Logging for `MI` debuggers
+
+The logging block with its defaults is as follows:
```
Comment (bobbrow) [inline docs/cpp/enable-logging-cpp.md]: There is no mention of `launch.json` before this section.  Consider mentioning that the logging block should be added to the configurations in `launch.json`

File: docs/cpp/enable-logging-cpp.md (hunk 2 above)
Comment (pieandcakes) [reply docs/cpp/enable-logging-cpp.md]: those were the only two where i thought it would be good to have the distinction.

**Comment by michelleangela** [comment]
@gregvanl, done reviewing this PR.

**Comment by gregvanl** [comment]
Published

**Pull request merged**